modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/web/internal/portlet/action/test/EditSegmentsEntryMVCRenderCommandTest.java

```diff
@@ -83,6 +94,65 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@Test
+	public void testErrorMessage() throws Exception {
```

Comment:
Following Liferay guidelines, the name of the tests should be the name of the method you are testing, in this case: getData, as we already have a testGetData, the name should be testGetDataWithErrorMessage.

Reply:
Understood, thank you